service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentUploadHelperTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.fpl.utils.DocumentUploadHelper;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
```

Comment:
The spring stuff can be removed if you want, as we have defined that everything else is mocked we just need to do the following instead

```java
    @Mock
    private IdamClient idamClient;

    @Mock
    private RequestData requestData;

    @InjectMocks
    private DocumentUploadHelper documentUploadHelper;
```

Reply:
done